### DISCUSSION TITLE
I am a novice,I try to repeat the model in a document, which is made with PKSIM, but there are some parameters I can't find, and some units I can't convert. What should I do? help me.

### DISCUSSION THREAD

**Discussion by Achineseman** [Open-Systems-Pharmacology/Forum#944] [Questions & Problems] [answered]
<img width="256" alt="屏幕截图 2021-07-31 003516" src="https://user-images.githubusercontent.com/81560924/127684847-f5ac90dc-8b4d-4e49-a338-a9b4efd22812.png">
<img width="525" alt="屏幕截图 2021-07-31 003700" src="https://user-images.githubusercontent.com/81560924/127684857-2e68e0be-08a8-42af-9c1d-a7eaa86369f1.png">
<img width="524" alt="屏幕截图 2021-07-31 003744" src="https://user-images.githubusercontent.com/81560924/127684868-2c4dac6b-1121-48ba-8034-9378fa39ca91.png">
<img width="960" alt="屏幕截图 2021-07-31 004352" src="https://user-images.githubusercontent.com/81560924/127685258-5e3b56ce-dadc-4e7d-9bef-78fe9a71e376.png">
<img width="960" alt="屏幕截图 2021-07-31 004432" src="https://user-images.githubusercontent.com/81560924/127685269-7946c4e6-41df-4ac0-a6d5-c45e65943c94.png">

**Comment by nasirkalam** [comment, marked as answer]
Dear  major plasma protein binding selection is available at 1st tab of
compound.
In table 4 values of CLint of CYP enzyme is calculated by retrograde model
of hepatic clearance. An additional 17.6% hepatic clearance was assigned in
2nd tab of compound. Then select both of these clearance mechanisms while
running simulations.
Fractional changes in liver cirrhosis were made by multiplying this
fractions with blood flow in individual file.
